src/index.js

```diff
@@ -60,7 +61,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(windowsPathRegExp, '\\\\')}';\n`;
```

Comment:
This looks like you are replacing 2 backslashes with 2 backslashes. Am I missing something?